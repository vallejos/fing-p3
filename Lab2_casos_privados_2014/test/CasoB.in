5
1 -1
2 3 -1
4 -1
4 -1 
-1
Flora 30 Fauna 20 Campo 15 -1 -1
Flora 25 Fauna 14 Campo 10 -1 -1
Flora 20 Fauna 10 Campo 8 -1 -1
Flora 15 Fauna 5 Campo 3 -1 -1
Flora 10 Campo 2 -1 -1
identificarColecciones 3 Flora Fauna Campo
identificarColecciones 3 Fauna Campo Flora 
identificarColecciones 3 Campo Flora Fauna
salir
