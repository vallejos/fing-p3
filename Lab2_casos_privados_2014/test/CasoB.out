Tarea 2 - 2014 - Programacion 3.

>0 -> Fauna
1 -> Flora
2 -> Campo
3 -> 
4 -> 
>0 -> Fauna
1 -> Flora
2 -> Campo
3 -> 
4 -> 
>0 -> Fauna
1 -> Flora
2 -> Campo
3 -> 
4 -> 
>FIN.
